src/App.js

```diff
@@ -11,6 +11,8 @@ function App() {
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/products" element={<Products />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/signup" element={<Signup />} />
       </Routes>
```

Comment:
CODE Suggestion:
Instead of adding routes in the App.js file, you could move it to a separate file say AppRoutes.js and then add the component here. This way, in the future, if the number of routes increases your code will look cleaner, readable easy to understand.

Reply:
done implemented